searchconsole/query.py

```diff
@@ -399,3 +399,53 @@ def to_dict(self):
     def to_dataframe(self):
         import pandas
         return pandas.DataFrame(self.rows)
+
+
+class IndexStatus:  
+    def __init__(self, api):
+        self.api = api
+        self.raw = {}
+
+    def __repr__(self):
+        return "<searchconsole.index_status(url={})>".format(self.api.url)
+    
+    def urls(self, *urls):
+        list_urls = list(urls)
+        
+        if 'sc-domain' in self.api.url:
+            matching_urls = [s for s in list_urls if self.api.url.split('sc-domain:')[1] in s]
+            if len(matching_urls) != len(list_urls):
+                raise ValueError("At least one of your URLs doesn't belong to the selected property.")
+        else: 
+            matching_urls = [s for s in list_urls if self.api.url in s]
+            if len(matching_urls) != len(list_urls):
+                raise ValueError("At least one of your URLs doesn't belong to the selected property.")
+        
+        self.raw['urls'] = list_urls
+        return self
+    
+    def get(self):
+        report = []
+        for url in self.raw['urls'] :
+            request = {
+                'siteUrl': self.api.url,
+                'inspectionUrl': url
+            }
+            result = self.api.account.service.urlInspection().index().inspect(body=request).execute()['inspectionResult']['indexStatusResult']
+            result['page'] = url
+            report.append(result)
+        
+        self.raw['results'] = report
```

Comment:
You shouldn't need this if you're passing in the report to `IndexStatusReport` as below.

Reply:
Good point. OOO is not my thing at all, so expect a lot of mistakes like this. I'm learning though :)

Reply:
No worries - happy to help!